### PR TITLE
env(backend): configure express environment

### PR DESCRIPTION
**Pull request by x566chen**
JSAP-16 added, plz check.

### CODE REVIEW

File: backend/app.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+require('dotenv').config()
+const express = require('express')
+const app = express()
+const port = process.env.PORT
+
+
```
Comment (matecserven) [inline backend/app.js]: Remove empty lines, leave only 1 between functions

File: backend/package.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{
+  "name": "backend",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
```
Comment (matecserven) [inline backend/package.json]: Write an npm start script to start our server

File: backend/app.js (hunk 1 above)
Comment (x566chen) [reply backend/app.js]: OK

File: backend/package.json (hunk 2 above)
Comment (x566chen) [reply backend/package.json]: OK

**Pull request merged**